pom.xml

```diff
@@ -5,7 +5,7 @@
 
     <groupId>io.micronaut.build</groupId>
     <artifactId>micronaut-maven-plugin</artifactId>
-    <version>3.4.1-SNAPSHOT</version>
+    <version>3.4.2-SNAPSHOT</version>
```

Comment:
Don't need to change this. The release plugin takes care of that.

Reply:
Yeah indeed. Sorted out I believe.